packages/rest/src/session/AbstractSession.ts

```diff
@@ -256,86 +235,107 @@ export abstract class AbstractSession {
     private buildSession(session: ISession): ISession {
         const populatedSession = session;
 
-
         // set protocol if not set
-        if (isNullOrUndefined(populatedSession.protocol)) {
+        if (populatedSession.protocol === undefined || populatedSession.protocol === null) {
```

Comment:
I've seen `isNullOrUndefined`, `property === null || property === undefined`, and `property == null` (all 3) in this PR so far.
Do we want to make them consistent (at least for the PR), or should we leave that for later? 😋

Reply:
I think we can do either or. We may want to make an effort in the future to clean everything up.

Reply:
I think that we should open a Git issue requesting that we make this consistent, and implement a solution in a technical debt story in some future sprint.

Reply:
Check this out!
https://github.com/zowe/imperative/issues/400

Feel free to edit/add more utilities and/or standards that could be implemented